http.go

```diff
@@ -162,3 +162,25 @@ func HealthCheckHandler(serviceName string) http.Handler {
 		w.Write([]byte(fmt.Sprintf("%s service is up.\n", serviceName)))
 	})
 }
+
+// ReqToObject un-marshals a request object body JSON into an object
+func ReqToObject(r *http.Request, obj interface{}) error {
```

Comment:
nit: Please follow the convention elsewhere in this code and most of go of spelling things out,  I don't see any logic to why we would exclude`uest` but not 'ject' .

Reply:
s/object/struct/

Reply:
Suggest `DeserializeJsonRequest` as a more accurate, go'ish name.